sirepo/uri_router.py

```diff
@@ -190,6 +203,22 @@ def uri_for_api(api_name, params=None, external=True):
     return res
 
 
+class _Route(PKDict):
```

Comment:
Document this: holds all route information. Document the known keys: func, cls, name, func_name, etc.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9